docs/content/concepts/automation/declarative-automation.mdx

```diff
@@ -75,6 +73,31 @@ Automation conditions describe the conditions under which an asset should be exe
     </tr>
   </thead>
   <tbody>
+    <tr>
+      <td>
+        <strong>AutomationCondition.on_cron(cron_schedule)</strong>
+      </td>
+      <td>
+        This condition will materialize an asset once per cron schedule tick,
+        after all of its parents have been updated since the tick
+      </td>
+      <td>
+        Regularly updating an asset without worrying about the specifics of how
+        its parents update
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>AutomationCondition.on_missing()</strong>
+      </td>
+      <td>
+        This condition will materialize an asset if it has never been
```

Comment:
Maybe:
`This condition will materialize an asset if all its dependencies have been updated, but the asset itself has not.`

Just so i understand, this means it will only be updated once and only once, and so really best suited for partitioned assets?

Reply:
exactly yep